doozer/doozerlib/build_info.py

```diff
@@ -579,7 +605,14 @@ def is_under_embargo(self) -> bool:
         if cn not in self._cache:
             with self.runtime.shared_build_status_detector() as bs_detector:
                 # determine if the image build is embargoed (or otherwise "private")
-                self._cache[cn] = len(bs_detector.find_embargoed_builds([self._brew_build_obj], [self.get_image_meta().candidate_brew_tag()])) > 0
+                self._cache[cn] = (
+                    len(
```

Comment:
Debatable formatting choice :sweat_smile: Do you think we have room for customization here?

Reply:
Yeah this is the case of the formatter trying to fix someting that was a poor readability decision in the first place - nested calls. imo it is okay to have these monstrosities, so that when we see one, we fix it by using vars like a sane person on a good day :D